spec/appliance_console/prompts_spec.rb

```diff
@@ -602,7 +608,17 @@ def expect_cls
   end
 
   def expect_heard(strs, check_eof = true)
-    strs = Array(strs).collect { |s| s == "" ? "\n" : s }.join
+    output_with_input.rewind
+    readline_output = output_with_input.read
+    if readline_output.empty?
+      strs = Array(strs).collect { |s| s == "" ? "\n" : s }.join
+    else
+      strs = Array(strs)
+      str1 = strs[0]
+      strs = strs[1..-1].collect { |s| s == "" ? "\n" : s }.join
+      first_output_correct = readline_output.include?(str1)
+      expect(first_output_correct).to be_truthy
+    end
```

Comment:
Quite simplistic 👍  Make all these changes. Except that  rewind returns an integer so can't combine rewind.read together. For menu prompts or Y/N prompts I think it's acceptable since user is not likely to move cursor to change one character. What do you think?